build/pkgs/fflas_ffpack/checksums.ini

```diff
@@ -1,5 +1,5 @@
-tarball=fflas_ffpack-VERSION.tar.bz2
-sha1=7c5faa81abc2b88ec24cec373b5e44cbaa7844dd
-md5=d8b7c113951a2a3f498a3aaadbe5620f
-cksum=3321469120
-upstream_url=https://github.com/linbox-team/fflas-ffpack/releases/download/vVERSION/fflas_ffpack-VERSION.tar.bz2
\ No newline at end of file
+tarball=fflas_ffpack-VERSION.tar.gz
+sha1=0beadd30b01dc5c890679aff73818a681513e82e
+md5=7b20fb95acb34d8d39fef2638cf2558e
+cksum=805176663
+upstream_url=https://github.com/mkoeppe/fflas-ffpack/releases/download/v2.5.0%2Bsage-2024-05-18b/fflas-ffpack-2.5.0.tar.gz
```

Comment:
Don't we have an "official" place under `https://github.com/sagemath/` to keep this tarball?

Can we keep the original url as commented out?

Reply:
easier to just revert the commit when we can, I'd say

Reply:
We may need to keep these "in-house" tarballs for long time...

Reply:
Perhaps we fork `fflas-ffpack` under `sagemath` and ...

Reply:
Well, I hope that my PRs that create releases will inspire upstream to make releases more often...

Reply:
> easier to just revert the commit when we can, I'd say

OK.

Reply:
> Well, I hope that my PRs that create releases will inspire upstream to make releases more often...

They do not seem to be active enough to be aware of your PRs. But let's see.

Your repo is OK for now.